src/scaffoldfitter/fitter.py

```diff
@@ -825,6 +829,28 @@ def getDataRMSAndMaximumProjectionError(self):
             del error
         return rmsErrorValue if (rmsResult == RESULT_OK) else None, maxErrorValue if (maxResult == RESULT_OK) else None
 
+    def getDataRMSAndMaximumProjectionErrorForGroup(self, groupName):
+        """
+        Get RMS and maximum error for the nodeset group with the given name.
```

Comment:
The values are calculated over the intersection of the group's data nodeset group and the active data group.